alacritty/src/renderer/mod.rs

```diff
@@ -88,7 +88,11 @@ impl std::error::Error for Error {
 
 impl Display for Error {
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
-        write!(f, "There was an error initializing the shaders: {}", self)
+        match self {
+            Error::ShaderCreation(err) => {
+                write!(f, "There was an error initializing the shaders: {}", err)
+            },
+        }
```

Comment:
My initial thought was that `if let` would be correct here, however if we do add new error variants we will need to add them to this match statement. So a match is the way to go here, to prevent things from going wrong in the future.